jdaviz/configs/imviz/helper.py

```diff
@@ -553,6 +560,11 @@ def link_image_data(app, link_type='pixels', wcs_fallback_scheme='pixels', wcs_u
         if not layer_is_image_data(data):
             continue
 
+        if data in data_already_linked:
+            # links already exist for this entry and we're not changing the type
+            # TODO: account for toggling wcs_use_affine
```

Comment:
Is this TODO for this PR or for "the future"?

Reply:
already handled in line 539, I'll remove the todo comment - thanks!